basic/src/test/java/org/jboss/hal/testsuite/test/configuration/requestcontroller/RequestControllerSubsystemTestCase.java

```diff
@@ -59,6 +59,7 @@ public static void afterClass() throws InterruptedException, TimeoutException, I
         }
     }
 
+    @Category(Standalone.class)
     @Test
     public void verifyActiveRequests() throws Exception {
```

Comment:
This test can be updated also for domain, the runtime attributes are specific for single server => you need to check it on the server level and not on profile level => all tests can remain shared for standalone and domain, just domain requires handling for single server in domain rather then whole profile.